### ISSUE TITLE
Handle HydroShare Collection resources

### ISSUE THREAD

**Issue by horsburgh** [cznethub/dsp#78]
**Describe the feature you'd like and what it will do**

Need to verify that the DSP can handle HydroShare collection resources. This includes:

1. Registration of existing HydroShare collection resources with the DSP (i.e., the collection already exists in HydroShare, but the user wants to register it with the DSP).
2. Creation of new HydroShare collection resources. This may need to be an alternative flow on the HydroShare submission form.

**Why is this feature important?**
A short description of the importance of this feature and what it will help you achieve.

Some users have already created HydroShare collection resources and need to register them with the DSP. Others may eventually want to create HydroShare collection resources and may want to do so through the DSP.

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...].

I can't do this now.

**Additional context**

**Comment by horsburgh** [comment]
@Maurier and @sblack-usu - we got another vote for this use case today in our call with the CZNet Data Managers. Can we consider two phases for this work:

Phase 1: Allow registration of existing collection resources from HydroShare with the Data Submission Portal so their metadata can be copied to the metadata catalog and so they become discoverable at discover.criticalzone.org.

Phase 2: Enable creation of HydroShare resources from the Data Submission Portal.

I'm thinking for Phase 1, we could use the existing HydroShare resource registration functionality. I haven't tried registering a HydroShare collection resource. Does that work already? If not, is that something that @pkdash could work on? For Phase 1, if registration of collection resources doesn't work already, no front end changes are needed, we just need the ability to register a HydroShare collection resource so it can become discoverable at discover.criticalzone.org.

**Comment by pkdash** [comment]
@horsburgh As we discussed today, I was able to register a hydroshare collection resource. Except the spatial coverage all other metadata was auto populated. Here I am leaving a note of changes to the UI we discussed to support collection resource:

- [x]  The file upload area needs to be disabled
- [x]  The list of resources should be displayed as links near the file upload area when viewing the collection resource metadata.
- [ ]  Metadata should not be editable once submitted. 
- [x]  If the collection resource has spatial coverage, it should be auto populated when registering a collection  resource.
- [x]  In the list of submissions, add a new field **Type** of the submission for any hydroshare to display either "Resource" or "Collection".
